AirCasting/SearchAndFollow/CompleteScreen/SearchSession.swift

```diff
@@ -0,0 +1,50 @@
+// Created by Lunar on 22/02/2022.
+//
+
+import Foundation
+
+struct SearchSession {
+    let name: String
+    let startTime: Date
+    let endTime: Date
+    let type: SessionType
+    let longitude: Double
+    let latitude: Double
+    let streams: [SearchSessionStream]
+    
+    struct SearchSessionStream: Identifiable {
+        var id: Int
+        let sensorPackageName: String?
+        let sensorName: String
+        let measurements: [SearchSessionMeasurement]
+        
+    }
+    
+    struct SearchSessionMeasurement {
+        let time: Date
+        let value: Double
+    }
+    
+    
+//    #if DEBUG
```

Comment:
Why it is here? and the //end below

Reply:
Cause it is a mocked session which in the future will be used only in the preview, but for now I am using it to pass session data to the view (it will be changed when we'll be able to pass session data from map view)